app/helpers/react_on_rails_helper.rb

```diff
@@ -189,15 +189,17 @@ def server_rendered_react_component_html(options, props, react_component_name, d
 
     wrapper_js = <<-JS
 (function() {
+#{ReactOnRails::ServerRenderingPool.before_render(react_component_name, props, options, request)}
 #{initialize_redux_stores}
   var props = #{props_string(props)};
   return ReactOnRails.serverRenderReactComponent({
-    name: '#{react_component_name}',
-    domNodeId: '#{dom_id}',
+    name: "#{react_component_name}",
```

Comment:
@martyphee This would not pass the linter. ;-) No double quotes in JS code. So please stick to single quotes. Remember, we're generating JS here, so flip the JS button in your head.